src/baseTypes.ts

```diff
@@ -0,0 +1,178 @@
+export enum NetworkId {
```

Comment:
What was the thinking for moving these here?

Reply:
Thank you for reviewing. There was a circular dependency of imports that failed tests.
Here is the background on this change:
https://github.com/OlympusDAO/olympus-frontend/pull/1311#discussion_r790459023

Reply:
wait a minute... this PR includes 1311?